wordpress-mobile/WordPress-Android
# Removed RtL support on SDK 17

Partially fixes #2278
Disables RtL layouts on api 17. RtL layouts are broken on API 17.

To test:

Api 17
1. Change locale to Hebrew.
2. Check that the layouts in the WordPress app are LtR

Api 18
1. Change locale to Hebrew.
2. Check that the layouts in the WordPress app are RtL
